src/main/resources/assets/viaproxy/language/ru_RU.properties

```diff
@@ -11,101 +11,108 @@
 # The display name of the language
 language.name=Русский (Россия)
 # The name of the author of the language file (Your name)
-language.author=Liticane (Weening)
+language.author=Felix14_v2 и Liticane (Weening)
```

Comment:
Please seperate multiple authors with a comma

Reply:
In Russian, to use comma as a separator, we need at least 3 elements. The exception is when we list some things in a technical documents (if so, we have to use colon before the names). I can use the comma anyways if you want, but imo we shouldn't do that